src/zepben/eas/client/opendss.py

```diff
@@ -0,0 +1,55 @@
+#  Copyright 2020 Zeppelin Bend Pty Ltd
+#
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+from dataclasses import dataclass
+from datetime import tzinfo
+from enum import Enum
+from typing import Union, Optional, List
+from zepben.eas.client.work_package import GeneratorConfig, TimePeriod, FixedTime
+
+__all__ = [
+    "OpenDssConfig",
+    "OpenDssModelState",
+    "GetOpenDssModelsFilterInput",
+    "Order",
+    "GetOpenDssModelsSortCriteriaInput"
+]
+
+@dataclass
```

Comment:
nit: unsure on the style guide for this repo, but generally you have 2 blank lines between module level class and function definitions

Reply:
fixed in f1f4c14d